vcr/cassette_yf.py

```diff
@@ -0,0 +1,7 @@
+def _loc_handle_generator(self, fn):
+    """Wraps a generator so that we're inside the cassette context for the
+    duration of the generator.
+    """
+    with self as cassette:
+        coroutine = fn(cassette)
```

Comment:
Any reason you don't: `yield from fn(cassette)`

Reply:
For readability mostly.

About test: this code is directly used by plenty tests so I'm not sure to follow what you'd like.

Reply:
unless trying to fit into 80 cols it's not worth breaking lines